Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -57,34 +59,35 @@ namespace gdjs {
       for (let i = 0; i < this._borderSprites.length; ++i) {
         this._spritesContainer.addChild(this._borderSprites[i]);
       }
-      this._alpha = this._spritesContainer.alpha;
+      this._wrapperContainer.addChild(this._spritesContainer);
+      this._alpha = this._wrapperContainer.alpha;
       runtimeScene
         .getLayer('')
         .getRenderer()
-        .addRendererObject(this._spritesContainer, runtimeObject.getZOrder());
+        .addRendererObject(this._wrapperContainer, runtimeObject.getZOrder());
     }
 
     getRendererObject() {
-      return this._spritesContainer;
+      return this._wrapperContainer;
     }
 
     ensureUpToDate() {
       if (this._spritesContainer.visible && this._wasRendered) {
         // Update the alpha of the container to make sure that it's applied.
         // If not done, the alpha will be back to full opaque when changing the color
         // of the object.
-        this._spritesContainer.alpha = this._alpha;
+        this._wrapperContainer.alpha = this._alpha;
         this._spritesContainer.cacheAsBitmap = true;
       }
       this._wasRendered = true;
     }
 
     updateOpacity(): void {
-      //TODO: Workaround a not working property in PIXI.js:
-      this._spritesContainer.alpha = this._spritesContainer.visible
-        ? this._object.opacity / 255
-        : 0;
-      this._alpha = this._spritesContainer.alpha;
+      // The alpha is updated on a wrapper around the sprite because a known bug
+      // in Pixi will create a flicker when cacheAsBitmap is set to true
+      // (see https://github.com/pixijs/pixijs/issues/4610)
+      this._wrapperContainer.alpha = this._object.opacity / 255;
+      this._alpha = this._wrapperContainer.alpha;
     }
 
     updateAngle(): void {
```

Comment:
While it seems to work just like this, I think we should update updateAngle, updatePosition and updateWidth/updateHeight to use the wrapperContainer.

The reason is that if we move only the position/rotation (and pivot point) of _spritesContainer, you can picture the wrapper container as being at position 0;0 in the world, and then "extending" infinitely to contain _spritesContainer (which could be at a large position like 3000;3000).
Instead, we'll do the reverse: move (and rotate) wrapperContainer. spritesContainer will stay inside wrapperContainer unmoved (so it will just follow whatever transform is applied to wrapperContainer).

I think this won't change anything, and the solution would work like this currently, but it seems more logic to have a wrapperContainer to be the one moving in the space and not a child of it (much like a div inside a div).
The current solution could have created an issue if we were to use the position of the wrapper (for example, if the scene would be fetching the coordinate of the `getRendererObject` to decide if an object must be hidden or not), but I think it's not the case (in my example, the scene is always using the position given by the object, not by the renderer - it's the object that is the source of truth).